pyOCD/target/target.py

```diff
@@ -15,11 +15,13 @@
  limitations under the License.
```

Comment:
Just committed a change to remove this file from the branch. Created a PR to update cmsis-svd with the latest Kinetis SVD files, including this one (looks like posborne just merged it :)